openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -173,7 +173,7 @@ def install():
         os.remove(filepath)
 
     icon = get_openpype_icon_filepath()
-    tab_menu_label = os.environ.get("AVALON_LABEL") or "AYON"
+    tab_menu_label = "AYON" if AYON_SERVER_ENABLED else "OpenPype"
```

Comment:
I think we should still consider `AVALON_LABEL`.
```
default_label = "AYON" if AYON_SERVER_ENABLED else "OpenPype"
tab_menu_label = os.environ.get("AVALON_LABEL") or default_label 
```

Reply:
So, should I uncommit my last commit ?